programmers/수박수박수박수박수박수.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+  let answer = "";
+  let addWord = "";
+  for (let i = 0; i < n; i++) {
+    addWord = i % 2 === 0 ? "수" : "박";
+    answer += addWord;
```

Comment:
```suggestion
    (i % 2) === 0 ? answer += "수" : answer += "박";
```
이러면 addWord라는 변수가 필요 없겠죠?